source/adminguide/virtual_machines/importing_unmanaging_vms.rst

```diff
@@ -401,6 +409,170 @@ Unmanaging Instance actions
+retrieval of detailed insights into the status of instances that are currently halted or stopped.
+
+**Request parameters**:
+
+.. parsed-literal::
+   - **zoneid** (Zone to which Instance will be imported)
+   - **host** (the host name or IP address of External Host)
+   - **username** (the username of External Host for authentication)
+   - **password** (the password of External Host for authentication)
+
+**Response**:
+
+.. parsed-literal::
+   - **name**
+   - **osdisplayname**
+   - **memory**
+   - **powerstate**
+   - **cpuCoresPerSocket**
+   - **cpunumber**
+   - **cpuspeed**
+   - **disk**
+      - **id**
+      - **capacity** (in bytes)
+      - **controller**
+      - **controllerunit**
+      - **imagepath**
+      - **position**
+   - **nic**
+      - **id**
+      - **macaddress**
+      - **networkname**
+      - **vlanid**
+      - **pcislot**
+      - **adaptertype** (when available)
+      - **ipaddress**
+
+
+importVm API
+~~~~~~~~~~~~
+
+importVm API serves as a tool for import of instances, identified by the provided name, from an external
+host into the CloudStack environment. Throughout this import process, steps are taken to ensure the secure and
+efficient transfer of disk images, formatted in the QCOW2 standard, from the external host to a designated storage pool
+within CloudStack.
+
+The conversion of existing disk images to a complete QCOW2 format is handled by the qemu-img utility, streamlining
+compatibility and enhancing performance within the CloudStack ecosystem. Administrators have the flexibility to
+configure the temporary storage location on the external host for the converted file, with the default location set to /tmp.
+
+**Request parameters**:
+
+.. parsed-literal::
+   - **zoneid** (Zone to which Instance will be imported)
+   - **host** (the host name or IP address of External Host)
+   - **username** (the username of External Host for authentication)
+   - **password** (the password of External Host for authentication)
+   - **importsource** (Import source should be external)
+   - **tmppath** (Temp Path on external host for disk image copy)
+   - **name** (Instance name on External Host)
+   - **displayname**
+   - **hostname**
+   - **account** (An optional account name for the Instance. Must be used with domainid parameter)
+   - **domainid** (An optional domain ID for the Instance. Must be used with account parameter)
+   - **projectid**
+   - **serviceofferingid**
+   - **nicnetworklist** (Map for NIC ID and corresponding Network UUID)
+   - **nicipaddresslist** (Map for NIC ID and corresponding IP address)
+   - **datadiskofferinglist** (Map for data disk ID and corresponding disk offering UUID)
+   - **details** (Map for Instance details)
+   - **forced** (If true, an Instance is imported despite some of its NIC's MAC addresses being already present)
+
+.. note:: The `forced` parameter is false by default and prevents importing an Instance which has a NIC containing a
```

Comment:
The formatting is incorrect 
![Screenshot 2023-11-13 at 2 36 29 PM](https://github.com/apache/cloudstack-documentation/assets/1401014/479823f9-336c-41ed-8d21-15c2a43a8ff6)